src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ### Bug Fixes
 - Fixed bug when creating a DataFrame with a dict of Series objects.
+- Fixed incorrect return type in `qcut` when given `Series` input and improved error checking logic.
+- Fixed bug when performing multiple DataFrameGroupBy apply/transform operations on the same DataFrame.
+
+### Behavior Changes
+- Given an input of type `Series`, `pd.qcut` always returns a `Series`.
```

Comment:
Same here.

Reply:
Moved this from 1.15.0a1 section to the latest section since it wasn't in the 1.15.0a1 release.